### ISSUE TITLE
[Sprint 6] Upgrade OSNet x0.25 to OSNet-AIN x1.0 (4x accuracy, ONNX ready)

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#6284]
**Impact:** HIGH (4x accuracy improvement) | **Effort:** 2-4h | **Source:** SOTA Vision Models Research (Report 07)

## Change

Replace OSNet x0.25 (2019, low accuracy) with OSNet-AIN x1.0 (verified ONNX, 4x better accuracy).

## Why OSNet-AIN x1.0 over ReIDMamba

The original recommendation was ReIDMamba, but research found:

* ReIDMamba is research-only (Nov 2025 arxiv), unknown license, untested ONNX export
* OSNet-AIN x1.0 has **verified ONNX weights** (`osnet_ain_x1_0_msmt17.onnx`)
* Same architecture family as current OSNet x0.25 (drop-in upgrade)
* MIT license, production-ready
* MSMT17 pretraining provides better cross-domain generalization for outdoor security cameras

## Performance

| Model | Market1501 R1 | Market1501 mAP | Size | ONNX |
| -- | -- | -- | -- | -- |
| OSNet x0.25 (current) | \~85% | \~70% | 2.5MB | Yes |
| **OSNet-AIN x1.0** | \~95% | \~86% | 9MB | **Yes (verified)** |

### 1\. Download ONNX model

```bash
# From torchreid / deep-person-reid
# Export or download osnet_ain_x1_0_msmt17.onnx
```

### 2\. Update Triton model config

Replace `ai/triton/model_repository/reid/` model with OSNet-AIN x1.0 ONNX.

### 3\. Update loader

Update `backend/services/osnet_loader.py` model path and verify input dimensions match (256x128).

### 4\. Verify embedding compatibility

OSNet-AIN x1.0 produces 512-dim embeddings (same as x0.25). Existing matching logic should work.

## Dark Horse: Pose2ID (CVPR 2025)

Training-free ReID using pose keypoints from existing YOLOv8n-Pose. Zero additional VRAM. Worth prototyping alongside this upgrade.

## References

* [deep-person-reid (torchreid)](<https://github.com/KaiyangZhou/deep-person-reid>)
* [Pose2ID (CVPR 2025)](<https://github.com/yuanc3/Pose2ID>)
